recipes/pgcgap/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "pgcgap" %}
+{% set version = "1.0.0" %}
+{% set sha256 = "71636dd95367d8879fa8d030f871db71be539aa01905701567479ea2b1d89a45" %}
+
+package:
+  name: '{{ name }}'
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/liaochenlanruo/pgcgap/archive/v{{ version }}.tar.gz
+  sha256: '{{ sha256 }}'
+
+build:
+  number: 0
+
+requirements:
+  build:
```

Comment:
Can you rename this to `host:`?

Reply:
Do you mean that I should change "build:" to "host:"?
Thanks for your help!

Reply:
Yes